src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -16,8 +16,8 @@
 public class HelpWindow extends UiPart<Stage> {
 
     public static final String USERGUIDE_URL =
-            "https://ay2122s2-cs2103-f11-2.github.io/tp/UserGuide.html";
-    private static final String HELP_MESSAGE = "Available functions on this version: \n\n";
+            "https://ay2122s2-cs2103-f11-2.github.io/tp/UserGuide.html#command-summary";
```

Comment:
Do we want to link them directly to the command summary ah? Cause what I intended was just to bring them to the full guide hahaha ! Up to you ah !

Reply:
Good point! I'll revert it back. Realised it's `Open User Guide in browser`.